pkgs/development/python-modules/ifcopenshell/default.nix

```diff
@@ -1,15 +1,21 @@
-{ lib, stdenv
+{ lib
+, stdenv
 , buildPythonPackage
 , fetchFromGitHub
 , gcc10
 , cmake
+, cgal
 , boost179
+, gmp
 , icu
 , swig
+, mpfr
 , pcre
 , opencascade-occt
 , opencollada
 , libxml2
+, hdf5
+, zlib
```

Comment:
Is there a convention on the order here? (it's a bit random for now). Should I order alphabetically?

Reply:
I ordered alphabetically.